# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../xorg/sparc-kbd.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

We do not need this conditional rename on sparc.

  - Rene Rebe <rene@exactcode.de>

--- ./programs/Xserver/hw/xfree86/os-support/linux/lnx_io.c.orig	2004-11-08 20:17:04.108288696 +0100
+++ ./programs/Xserver/hw/xfree86/os-support/linux/lnx_io.c	2004-11-08 20:19:01.000000000 +0100
@@ -74,7 +74,7 @@
 #endif
 
 /* Deal with spurious kernel header change */
-#if defined(LINUX_VERSION_CODE) && defined(KERNEL_VERSION)
+#if !defined(__sparc__) && defined(LINUX_VERSION_CODE) && defined(KERNEL_VERSION)
 # if LINUX_VERSION_CODE >= KERNEL_VERSION(2,5,42)
 #  define rate period
 # endif
--- ./programs/Xserver/hw/xfree86/os-support/linux/lnx_kbd.c.orig	2004-11-08 20:19:39.792621080 +0100
+++ ./programs/Xserver/hw/xfree86/os-support/linux/lnx_kbd.c	2004-11-08 20:20:06.000000000 +0100
@@ -101,7 +101,7 @@
 #endif
 
 /* Deal with spurious kernel header change */
-#if defined(LINUX_VERSION_CODE) && defined(KERNEL_VERSION)
+#if !defined(__sparc__) && defined(LINUX_VERSION_CODE) && defined(KERNEL_VERSION)
 # if LINUX_VERSION_CODE >= KERNEL_VERSION(2,5,42)
 #  define rate period
 # endif
